user-info-provider/README.md

```diff
@@ -135,3 +135,57 @@ This mapper produces the following standardized metadata JSON-structure
 }
 ```
 
+## Configuration based on role
+There is now a possibility to hide the person tag in the claim based on the role attribute which the user has. If this
+needs to be used please add the professional roles which need to be checked in the mapper and select the negate output
+accordingly.
```

Comment:
I don't really get what this is meaning :sweat_smile: maybe after review I will get it. But now I think the following:
- What is a "person tag"?
- What does this "professional roles" mean?
- "select the negate output **accordingly**" - accordingly to what context?